rviz_default_plugins/include/rviz_default_plugins/displays/image/ros_image_texture.hpp

```diff
@@ -136,6 +136,8 @@ class ROSImageTexture : public ROSImageTextureIface
 
   uint32_t width_;
   uint32_t height_;
+  uint32_t stride_;
+  std::shared_ptr<std::vector<uint8_t>> bufferptr_;
```

Comment:
This is going to be a problem for Galactic, since it changes the ABI.  We'll have to think about another way to do this.

Reply:
These two variables need to have the scope of the object so another option would be to make them global variables and reset the bufferptr__ in the destructor if the object